### DISCUSSION TITLE
Thermostat / Heat - ICON COLOR (Optional) split separately, Child lock

### DISCUSSION THREAD

**Discussion by elemarek** [Blackymas/NSPanel_HA_Blueprint#572] [Ideas]
1. Option to set a separate color for the thermostat icon and the status when the heating is active. Thermostat / Heat - ICON COLOR (Optional) split separately.

2. Be able to lock the screen. Child lock or something similar. Entering some code or holding down some combination of virtual buttons.

Thanks.

**Comment by edwardtfn** [comment]
I believe the item 1 was already implemented, right?
For item 2 I've moved to a Enhancement request:
- #1727
